pymbolic/primitives.py

```diff
@@ -381,6 +381,7 @@ def __rrshift__(self, other):
 
     def __inv__(self):
         return BitwiseNot(self)
+    __invert__ = __inv__
```

Comment:
```suggestion
    def __invert__(self):
        return BitwiseNot(self)
```

Reply:
Looking through the docs, I can't even find any evidence that `__inv_` was even ever a thing.